blynk_ncp/src/zephyr_lib.c

```diff
@@ -50,6 +50,10 @@ K_THREAD_STACK_DEFINE(ncp_stack_area, BLYNK_THERAD_STACK_SIZE);
 static struct k_thread ncp_thread_data;
 static k_tid_t ncp_tid;
 
+static uint8_t ncp_state = BLYNK_STATE_NOT_INITIALIZED;
```

Comment:
`ncp_state` should return to the `BLYNK_STATE_NOT_INITIALIZED` in some cases... please check it

Reply:
it drops to BLYNK_STATE_NOT_INITIALIZED at each init now